tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,22 +6,7 @@
    "source": [
```

Comment:
<p>This part needs major refactoring because the way to import and instantiate the simulators have changed a lot. We can discuss more in details later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Looking at the <a href="https://qiskit.org/documentation/apidoc/aer_provider.html" target="_blank">Aer Provider API reference</a> it seems the current way of simulating is with <code>AerSimulator</code>, given that <code>QasmSimulator</code>, <code>StatevectorSimulator</code> and <code>UnitarySimulator</code> are listed as "legacy simulator backends".</p><p> </p><p>What I'm not sure about is if there's any difference between using <code>Aer.get_backend("aer_simulator")</code>	and using <code>AerSimulator()</code>	.</p><p> </p><p>Looking at this https://quantumcomputing.stackexchange.com/questions/21667/difference-between-aer-get-backendaer-simulator-and-aersimulator even Matthew says they do the same and did not show a very clear preference. Only says that the <code>get_backend</code> option made more sense for remote backends.</p>